kafka-ui-api/src/main/java/com/provectus/kafka/ui/kafka/KafkaService.java

```diff
@@ -72,6 +77,27 @@ public void loadClusterMetrics(KafkaCluster kafkaCluster) {
         );
     }
 
+    public Mono<Topic> updateTopic(KafkaCluster cluster, String topicName, TopicFormData topicFormData, Integer id) {
+        ConfigResource topicCR = new ConfigResource(ConfigResource.Type.TOPIC, topicName);
+        List<ConfigResource> brokerCR = Collections.singletonList(new ConfigResource(ConfigResource.Type.BROKER, id.toString()));
+        return ClusterUtil.toMono(cluster.getAdminClient().describeConfigs(brokerCR).all())
+                .flatMap(c -> {
+                    if (oldClusterVersion(c, cluster)) {
```

Comment:
we could cache cluster version on adminclient cache level.

Reply:
cluster supported commands enum added to KafkaCluster object